swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self,inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory 
+    
+    def add(self,new_item):
+        self.inventory.append(new_item)
+        return new_item 
+    
+    def remove(self,item):
+        try:
+            self.inventory.remove(item)
+            return item 
+        except:
+            return False
+        
+    def get_by_id(self,id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None 
+    
+    def swap_items(self,other_vendor,my_item,their_item):
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        else:
+            self.inventory.remove(my_item)
+            other_vendor.inventory.append(my_item)
+            self.inventory.append(their_item)
+            other_vendor.inventory.remove(their_item)
+            return True  
+        
+    def swap_first_item(self,other_vendor):
+        if (not self.inventory) or (not other_vendor.inventory):
+            return False
+        my_item = self.inventory.pop(0)
+        their_item = other_vendor.inventory.pop(0)
+        self.inventory.append(their_item)
+        other_vendor.inventory.append(my_item)
```

Comment:
This would be a great place to reuse the `swap_items` method you created earlier.